kvbc/test/order_test.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2018 VMware, all rights reserved
+/**
+ * Test the Sliver class.
+ *
+ * While these tests check that Sliver "works", the most interesting test is to
+ * run this under valgrind, and make sure that it doesn't find any
+ * leaks/double-frees/etc. `valgrind --leak-check=full test/SliverTests`
+ */
```

Comment:
I think we should change that comment to reflect the tests in this file.

Reply:
agree